new-pipeline/src/dashboards.js

```diff
@@ -669,10 +669,10 @@ function getHumanReadableOptions(filterName, options) {
           (version <= latestNightlyVersion - 1 ? goodOptions : badOptions)
```

Comment:
Should we also instead put all aurora > 53 in the badlist, and nix it from the "latest in each channel" bit, below?

Or is that a story for another time?